service/dps/utils/src/main/java/eu/europeana/cloud/service/dps/service/utils/validation/DpsTaskValidator.java

```diff
@@ -105,6 +107,14 @@ public DpsTaskValidator withDataEntry(String entryName, InputDataValueType conte
         return this;
     }
 
+
+    public DpsTaskValidator withAllowedValuesParam(String paramName, Object allowedValues) {
```

Comment:
Please remove it if not used. (I got why you have it in the first and I think we shoud use it)